account_analytic_project/__init__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: UTF-8 -*-
```

Comment:
Please use simple headers in all py files following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3